http4s/src/test/scala/io/github/jkobejs/zio/google/cloud/oauth2/http4s/webserver/Fixtures.scala

```diff
@@ -0,0 +1,38 @@
+package io.github.jkobejs.zio.google.cloud.oauth2.webserver
```

Comment:
Why is this file in http4s package? There's nothing http4s specific here so maybe we should move it to "test commons" package?

Reply:
It is left over from migration, I'll remove it.